py/picca/xcf.py

```diff
@@ -277,22 +278,23 @@ def metal_dmat(pix,abs_igm="SiII(1526)"):
 
                 rp_abs = (rd_abs-rq)*sp.cos(ang/2)
                 rt_abs = (rdm_abs+rqm)*sp.sin(ang/2)
+                zwe = ((1+zd_abs)/(1+zref))**(alpha_abs[abs_igm]-1.)
```

Comment:
Sory to annoy you but can you use `1.` instead of `1`. In python 2, division can be impredictable with integers.